documentation/sphinx/source/release-notes/release-notes-630.rst

```diff
@@ -8,7 +8,7 @@ Release Notes
 ======
 * Added a hint field in the trace event when all replicas of some data are lost. `(PR #4209) <https://github.com/apple/foundationdb/pull/4209>`_
 * Rewrote SQLite injected fault handling. `(PR #4212) <https://github.com/apple/foundationdb/pull/4212>`_
-* Add a SevWarnAlways trace line to help debug a rare failure. `(PR #4214) <https://github.com/apple/foundationdb/pull/4214>`_
+* Add a SevWarnAlways trace line to help diagnose a rare situation that may block excluding a storage server. `(PR #4214) <https://github.com/apple/foundationdb/pull/4214>`_
```

Comment:
If you look at past release notes, they typically include only the subset of changes that has an impact on the community. Those items that are included are written so as to describe the impact of the change to the users rather than in terms of implementation details. This helps keep the noise low and hopefully makes it easier for someone to understand what they are getting with a release.

If you look at this change this change and some of the others here (e.g. adding unit test infrastructure, adding debug tools to runtime image, fixing simulation bugs, etc.), these aren't going to be relevant to people downloading and running our software normally. I think these should all be removed from the release notes, at least with respect to the existing document published on the website. If it's desired that we also have a complete change log somewhere, then that might warrant a separate document.